apps/docs/content/docs/introduction.mdx

```diff
@@ -8,26 +8,22 @@ order: 0
 
 Welcome to the tldraw developer docs.
 
-Here at tldraw, we make two things: a very good multiplayer whiteboard (at [tldraw.com](https://www.tldraw.com)) and the [open source](https://github.com/tldraw/tldraw) libraries used to create that product. This page is meant to provide documentation and reference for those open source libraries.
+Here at tldraw, we make two things: a very good multiplayer whiteboard (at [tldraw.com](https://tldraw.com)), and the [open source library](https://github.com/tldraw/tldraw) that powers it. This page provides documentation and reference for that open source library.
```

Comment:
I think we should adopt the language of '**open source library**' instead of '**open source libraries**'.

I think it'll be much easier + simpler to talk about it that way.
The 'tldraw library' can include all of tldraw's packages, and I think that's ok.

Also I tweaked the wording here to make it more intentional 💪